src/docfx/lib/log/Error.cs

```diff
@@ -13,7 +13,7 @@ internal class Error
     {
         public static readonly IEqualityComparer<Error> Comparer = new EqualityComparer();
 
-        public ErrorLevel Level { get; private set; }
+        public ErrorLevel Level { get; set; }
```

Comment:
@msftbot hold this pr for another 30 minutes so that @yufeih has a chance to consider my feedback